styles.css

```diff
@@ -1,3 +1,335 @@
-* {
-    font-size: 1.5rem;
+li a {
+    font-family: "Bakbak One", serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    list-style: none;
+    text-decoration: none;
+    color: #fff;
+}
+
+.toggle-button {
+    padding: 0;
+    border: 0;
+    height: 48px;
+    background-color: transparent;
+    cursor: pointer;
+}
+
+.discord-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #fff;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    width: 147px;
+    height: 48px;
+    background-color: #141B22;
+    cursor: pointer;
+}
+.discord-button img {
+    display: block;
+}
+
+.connect-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #000;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    height: 48px;
+    width: 149px;
+    background: #21E786;
+    cursor: pointer;
+}
+connect-button img {
+    display: block;
+}
+
+.breadcrumbs a {
+    font-size: 14px;
+    color: #000;
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    line-height: 20px;
+}
+
+h1,
+h2 {
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+h3 {
+    font-family: "Bakbak One", serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+p {
+    font-family: Poppins, serif;
+    font-weight: 400;
+    font-size: 18px;
+    color: #C2C3C5;
+}
+
+.list-term {
+    font-family: "Bakbak One", serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.list-description {
+    font-family: "Bakbak One", serif;
+    font-size: 56px;
+    font-weight: 400;
+    line-height: 68px;
+    color: #fff;
+    text-shadow: 0 4px 16px #FFFFFF66;
+}
+
+.list-item-number {
+    font-family: "Bakbak One", serif;
+    font-weight: 400;
+    font-size: 80px;
+    line-height: 96px;
+    color: #21E786;
+    opacity: 30%;
+}
+
+.list-item-sub {
+    font-family: Poppins, serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+}
+
+body {
```

Comment:
**Styles for the body should be written at the very beginning of the file:**

Placing the styles for the body element at the top of your CSS file ensures that global styles are defined first. Since the body element wraps the entire content of your webpage, its styles often form the foundation for the rest of the page's design. By defining these styles at the start, you set a clear baseline for how the entire page should look and function. This practice also helps in maintaining a logical flow in your CSS, making it easier to understand and manage.